pkg/math_utils/utils.go

```diff
@@ -0,0 +1,21 @@
+package math_utils
+
+import (
+	"math/big"
+
+	"github.com/pkg/errors"
+)
+
```

Comment:
func MathUtilsError(err error) error {
	return errors.Wrapf(err, "MathUtils error")
}

func ErrDivZero(n *big.Int, m *big.Int, p *big.Int) error {
	return MathUtilsError(errors.Errorf("%d in DivMod(n, %d, p) can't be zero", m, m))
}